assets/prefabs/buildings/Smithy.prefab

```diff
@@ -0,0 +1,10 @@
+{
+    "GenericBuilding" : {
+     "name" : "Smithy",
+     "templateNames" : ["HjerlHede:SmithTemplate"],
+     "zone" : "COMMERCIAL",
+     "minSize" : [15, 15],
+     "maxSize" : [15, 15],
+     "isEntity" : true
```

Comment:
The smith could consume coal and iron to produce some goods. maybe we'll need one or the other iteration on this to figure out which items to use here, or how this can be extended in general...